### ISSUE TITLE
specify models to run in the input settings

### ISSUE THREAD

**Issue by shapiromatron** [shapiromatron/bmds-server#1]
I’ll add a new option for “models”, which will be required. To get the tool with outputs in the current format, you would set to “default”, otherwise you can set to an array of objects. Each object will be a model name, and I’ll have to add a way to specify custom model settings for that model. For example, you should be able to do something like this:
```json
{
 "models": [
    {"name": "Polynomial", "model_settings": {"degree": 2}},
    {"name": "Polynomial", "model_settings": {"degree": 3}},
    {"name": "Polynomial", "model_settings": {"degree": 4}}
  ],
}

```
Or this:

```json
{
 "models": "default"
}
```

**Comment by shapiromatron** [comment]
Partially implemented (just model-name) in 039af04a29e7c5d871f90acde7b054e8f59e3e00

**Comment by shapiromatron** [comment]
Fully implemented!
